microsoft/testsuites/kata_conformance/create_pod.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myapp-pod
+spec:
+  runtimeClassName: kata-qemu
```

Comment:
This is the runtime class we create with runtime_class.yaml. It can have any name i guess. I followed which is mentioned in the doc shared by Vivek at early stage.